test/jte-hotreload-test/pom.xml

```diff
@@ -15,6 +15,7 @@
         <maven.deploy.skip>true</maven.deploy.skip>
         <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
         <jacoco.version>0.8.5</jacoco.version>
+	    <project.build.outputTimestamp>2023-01-01T00:00:00Z</project.build.outputTimestamp>
```

Comment:
Everything in `test` are internal integration test projects that are never released (`<maven.deploy.skip>true</maven.deploy.skip>`).

They do not need these changes.

Reply:
I think we should make everything reproducible if we now spend some time anyway.

Reply:
Seems like a lot of boilerplate for no real world impact.

Reply:
The timestamp doesn’t need to change on every build. Only for release. 

It doesn’t even have to be the commit time to be reproducible.

It would be ridiculous IMO to change the parent pom on every commit.